File: Semana 1 - Fundamentos/03 - Calculadora Simples/03_Calculadora_Simples.js
// Description: Simple calculator that adds two numbers
const readline = require('readline');

function main() {
    const rl = readline.createInterface({
        input: process.stdin,
        output: process.stdout
    });
    
    rl.question("Digite o primeiro numero: ", (num1) => {
        rl.question("Digite o segundo numero: ", (num2) => {
            const soma = parseFloat(num1) + parseFloat(num2);
            console.log(`A soma de ${num1} e ${num2} é: ${soma}`);
            rl.close();
        });
    });
}

main();

File: 01 - Trilha interativa de desafios de programação para iniciantes/Semana 1 - Fundamentos/03 - Calculadora Simples/03_Calculadora_Simples.py
# Description: Simple calculator that adds two numbers
def main():

    num1 = float(input("digite um numero: "))
    num2 = float(input("digite outro numero: "))
    soma = num1 + num2
    print(f"A soma de {num1} e {num2} é: {soma}")

if __name__ == "__main__":
    main()